specification/riskiq/Easm/models.tsp

```diff
@@ -0,0 +1,1645 @@
+  totalElements?: int64;
+
+  @doc("The link to access the next page of results.  Not set if at the end of the result set.")
+  nextLink?: string;
+
+  @doc("The items in the current page of results.")
+  value?: DataConnection[];
+}
+
+@doc("The properties required to establish connection to a particular service")
+model DataConnectionProperties {}
+
+#suppress "@azure-tools/typespec-azure-core/documentation-required"
+@discriminator("kind")
+model DataConnectionData {
+  @doc("The name of data connection")
+  name?: string;
+
+  @doc("The type of data the data connection will transfer.")
+  content?: DataConnectionContent;
+
+  @doc("The rate at which the data connection will receive updates.")
+  frequency?: DataConnectionFrequency;
+
+  @doc("The day to update the data connection on. (1-7 for weekly, 1-31 for monthly)")
+  frequencyOffset?: int32;
+}
+
+#suppress "@azure-tools/typespec-azure-core/documentation-required"
+@discriminator("kind")
+@resource("dataConnections")
+model DataConnection {
+  @doc("The system generated unique id for the resource.")
+  id?: string;
+
+  @doc("The caller provided unique name for the resource.")
+  @key("dataConnectionName")
+  @visibility("read")
+  name: string;
+
+  @doc("The name that can be used for display purposes.")
+  displayName?: string;
+
+  @doc("The type of data the data connection will transfer")
+  content?: DataConnectionContent;
+
+  @doc("The date the data connection was created.")
+  createdDate?: utcDateTime;
+
+  @doc("The rate at which the data connection will receive updates.")
+  frequency?: DataConnectionFrequency;
+
+  @doc("The day to update the data connection on.")
+  frequencyOffset?: int32;
+
+  @doc("The date the data connection was last updated.")
+  updatedDate?: utcDateTime;
+
+  @doc("The date the data connection was last updated by user.")
+  userUpdatedAt?: utcDateTime;
+
+  @doc("An indicator of whether the data connection is active.")
+  active?: boolean;
+
+  @doc("A message that specifies details about data connection if inactive.")
+  inactiveMessage?: string;
+}
+
+#suppress "@azure-tools/typespec-azure-core/documentation-required"
+model DependentResource {
+  md5?: string;
+  responseBodySize?: int64;
+  firstSeen?: utcDateTime;
+  lastSeen?: utcDateTime;
+  count?: int64;
+  firstSeenCrawlGuid?: string;
+  firstSeenPageGuid?: string;
+  firstSeenResourceGuid?: string;
+  lastSeenCrawlGuid?: string;
+  lastSeenPageGuid?: string;
+  lastSeenResourceGuid?: string;
+  responseBodyMinhash?: int32[];
+  contentType?: string;
+  sha256?: string;
+  sha384?: string;
+  sha512?: string;
+  url?: url;
+  cached?: boolean;
+  sriChecks?: SubResourceIntegrityCheck[];
+  host?: string;
+  lastObservedViolation?: utcDateTime;
+  lastObservedValidation?: utcDateTime;
+  lastObservedActualSriHash?: string;
+  lastObservedExpectedSriHash?: string;
+}
+
+#suppress "@azure-tools/typespec-azure-core/documentation-required"
+model DiscoGroupPageResult {
```

Comment:
I thought we had decided to use "Discovery" rather than "Disco".

Reply:
We have been using this naming pattern for the paths, and this version has been approved. I would keep this model naming pattern to keep it consistent with the path